src/v2/connectors/repository/invoice-accounts.js

```diff
@@ -0,0 +1,45 @@
+const { InvoiceAccount } = require('../bookshelf');
+
+/**
+ * Find single InvoiceAccount with relations by ID
+ * @param {String} id
+ * @return {Promise<Object>}
+ */
+const findOne = async id => {
+  const result = await InvoiceAccount
+    .forge({ invoice_account_id: id })
```

Comment:
Should this be consistent with the water service as camel case? .forge({ invoiceAccountId: id })

Reply:
Done